CodeGeneration/Sources/SyntaxSupport/AttributeNodes.swift

```diff
@@ -0,0 +1,892 @@
+//// Automatically Generated From AttributeNodes.swift.gyb.
+//// Do Not Edit Directly!
```

Comment:
Presumably these comments are now outdated?

Reply:
Fixed in https://github.com/apple/swift-syntax/pull/1396